src/views/Home.vue

```diff
@@ -6,36 +6,35 @@
     </div>
 
     <div class="main">
-      <ModuleNav />
-<!--      <section class="left-section">-->
-<!--        <router-link :to="{-->
-<!--          name: 'Module',-->
-<!--          params: { id: 1}-->
-<!--        }">-->
-<!--          <h1 class="module">Module 1</h1>-->
-<!--        </router-link>-->
+      <div class="left-section">
+      <router-link class="module" :to="{
+       name: 'StudentFeedback',
+       params: { id: 1 }
+     }">
+         <button class="module-btn">Module 1</button>
+     </router-link>
 
-<!--        <router-link :to="{-->
-<!--          name: 'Module',-->
-<!--          params: { id: 2}-->
-<!--        }">-->
-<!--          <h1 class="module">Module 2</h1>-->
-<!--        </router-link>-->
+     <router-link class="module" :to="{
+       name: 'StudentFeedback',
+       params: { id: 2 }
+     }">
+       <button class="module-btn">Module 2</button>
+     </router-link>
 
-<!--        <router-link :to="{-->
-<!--          name: 'Module',-->
-<!--          params: { id: 3}-->
-<!--        }">-->
-<!--          <h1 class="module">Module 3</h1>-->
-<!--        </router-link>-->
+     <router-link class="module" :to="{
+       name: 'StudentFeedback',
+       params: { id: 3 }
+     }">
+       <button class="module-btn">Module 3</button>
+     </router-link>
 
-<!--        <router-link :to="{-->
-<!--          name: 'Module',-->
-<!--          params: { id: 4}-->
-<!--        }">-->
-<!--          <h1 class="module">Module 4</h1>-->
-<!--        </router-link>-->
-<!--      </section>-->
+     <router-link class="module" :to="{
+       name: 'StudentFeedback',
+       params: { id: 4 }
+     }">
+       <button class="module-btn">Module 4</button>
+     </router-link>
+   </div>
```

Comment:
Now that we have the module number, we can use a v-for to iterate through these modules (to build up each link). in a future iteration.